src/car.js

```diff
@@ -0,0 +1,36 @@
+export function runRace(cars, count, resultContainer) {
+  resultContainer.innerHTML = '';
+  for (let i = 0; i < count; i += 1) {
+    let statusString = '';
+    cars.forEach((car) => {
+      car.go();
+      statusString += car.getStatus();
+      statusString += '<br>';
+    });
+    statusString += '<br>';
+    resultContainer.innerHTML += statusString;
+  }
+  const top = Math.max(...cars.map((car) => car.pos));
+  resultContainer.innerHTML += `최종 우승자: ${cars.filter((car) => (car.pos === top)).map((car) => car.name).join(', ')}`;
+}
+
+function getPath(pos) {
+  let result = '';
+  for (let i = 0; i < pos; i += 1) {
+    result += '-';
+  }
+  return result;
```

Comment:
```suggestion
  return String().padEnd(pos, '-');
```
이 라인에 코멘트 달았었는데 날아가서 추가합니다 ㅠㅠ
String에 [이런 메소드](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/String/padEnd)가 있더라고요! 대신 써봐도 좋을 것 같습니다.

Reply:
@hyo-choi 재밌는 메서드네요. 배워갑니다.